app/services/integrations/aggregator/taxes/invoices/payload.rb

```diff
@@ -47,6 +47,7 @@ def fee_item(fee)
             mapped_item ||= empty_struct
 
             {
+              'item_key' => fee.item_key,
```

Comment:
I think this does not need to be a new key right? anrok will use item_id for uniqueness and fee.item_key is always unique?

Reply:
I'd rather not touch `item_id` for now because it's used in the frontend. `item_id` is usually unique but not necessarily. If you have 2 charge fees relying on the same billable metric, the `item_id` will be the same.
This is what we wanted to fix here: https://github.com/getlago/lago-api/pull/2842

But sometimes, the fee isn't persisted (here or when createing a one-off invoice from the frontend. The frontend expects the item_id to be as defined in `Fee.item_id`.

So I'm introducing this new key. Ideally, both should be merged at some point to have something the frontend can use (not like `object_id`).

Reply:
okay, that all sounds really reasonable to me :)